<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QDialog" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #A4A8A6;
</string>
  </property>
  <widget class="QLabel" name="label_subject">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>280</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-weight: bold;
font-size: 18px;
border: 1px solid;
background-color: #06A310;</string>
   </property>
   <property name="text">
    <string>SUBJECT : &lt;name&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>680</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font-size : 15px;
background-color : #05BF11;
color: &quot;white&quot;;
font-weight: bold;
border: 1px solid;
border-radius: 4px;
border-color: #95FF93;
}

 QPushButton:hover
{
background-color : green;
}</string>
   </property>
   <property name="text">
    <string>&lt;&lt; BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>680</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font-size : 15px;
background-color : #05BF11;
color: &quot;white&quot;;
font-weight: bold;
border: 1px solid;
border-radius: 4px;
border-color: #95FF93;
}

 QPushButton:hover
{
background-color : green;
}</string>
   </property>
   <property name="text">
    <string>Add Question</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_question">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>381</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFF5B2;
border-radius:5px;
border: 1px groove;
border-color: #FFCF38;
font-size: 15px;</string>
   </property>
   <property name="placeholderText">
    <string>Write the question here</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_answer">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>520</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFF5B2;
border-radius:5px;
border: 1px groove;
border-color: #FFCF38;
font-size: 15px;</string>
   </property>
   <property name="placeholderText">
    <string>Write the answer here</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_question">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/question icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_answer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/answer icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>640</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FF0000;
font-size: 12px;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>ERROR LABEL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>-10</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e9e9e9;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/questions_answers.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e9e9e9;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_bg</zorder>
  <zorder>label</zorder>
  <zorder>label_subject</zorder>
  <zorder>btn_back</zorder>
  <zorder>btn_add</zorder>
  <zorder>txt_question</zorder>
  <zorder>txt_answer</zorder>
  <zorder>label_question</zorder>
  <zorder>label_answer</zorder>
  <zorder>label_error</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
